src/coreclr/src/jit/emit.cpp

```diff
@@ -1131,18 +1131,16 @@ float emitter::insEvaluateExecutionCost(instrDesc* id)
 //              if we return these are updated with default values
 //
 // Notes:
-//     When validating that the PerfScore handles every instruction.
-//     the #if 0 block is changed into a #ifdef DEBUG
-//     We will print the instruction and instruction group
+//     We print the instruction and instruction group
 //     and instead of returning we will assert
 //
-//     Otherwise we will return default latencies of 1 cycle.
+//     Remove the assert and we return default latencies of 1 cycle.
```

Comment:
I think that this should be deleted, and the above comment should be clarified to say that it asserts in a DEBUG build, and returns a default latency of 1 cycle otherwise.

Reply:
OK